src/openapi.json

```diff
@@ -2362,7 +2381,7 @@
 								"schema": {
 									"type": "array",
 									"items": {
-										"$ref": "#/components/schemas/Source"
+										"$ref": "#/components/schemas/SourceBundle"
```

Comment:
This is saying it will be an array of source bundles -- check the signature of other endpoints to make sure you're updating it to what would be expected.